auth/auth.go

```diff
@@ -96,6 +96,8 @@ func GetTokenFromBrowser(ctx context.Context, conf *oauth2.Config) (*oauth2.Toke
 			return nil, fmt.Errorf("failed to exchange code for token: %w", err)
 		}
 
+		// TODO: save the token somewhere
```

Comment:
Wasn't too sure if we wanted to keep this here or not

Reply:
Haha I added a comment then deleted it cuz I thought it was too nitpicky lol. Let's delete this line!